.gitignore

```diff
@@ -31,3 +31,5 @@ docs-prerelease.json
 docs.json
 .dub/
 dpl-docs/dpl-docs
+
+/d.tag
```

Comment:
Also, as mentioned in #870, $TMP is usually a global resource and should be used carefully. Early versions of rdmd used to have this problem - after one user used rdmd once, rdmd was not usable by any other user on the system.